### PR TITLE
Show variance percentage of principal components in axis labels

### LINKED ISSUE #168: ENH: Percentage of Variance Explained for PCA Axis Labels
It would be nice if instead of only "PC1" and "PC2", `autoplot` would automatically show axis labels like "PC1 (55.3 %)" and "PC2 (27.5%)".

### PR DESCRIPTION
**Pull request by lselzer**
I think this closes #168 
image

### CODE REVIEW

**Comment by lselzer** [comment]
I forgot to mention that I added an example showing how to plot the component 2 and 3. Which I think closes #164. I don't know if it worth to add it to the vignette.

File: R/fortify_stats.R [hunk 1]
```diff
@@ -262,6 +269,8 @@ autoplot.pca_common <- function(object, data = NULL,
     lam <- lam * sqrt(nrow(plot.data))
 
   } else if (is_derived_from(object, 'princomp')) {
+
+    ve <- object$sdev^2/sum(object$sdev^2)
```
Comment (terrytangyuan) [inline R/fortify_stats.R]: Could you put some spaces around the operations?

**Comment by terrytangyuan** [comment]
Thanks! I think an example is sufficient. You can modify existing PCA plotting vignettes if you want (add a line or two to show this functionality is available).

**Comment by terrytangyuan** [comment]
Could you fix the lint issues as well? Thanks!

**Comment by terrytangyuan** [comment]
Issue regarding `gglagplot (@test-tslib.R#103)` can be ignored, which will be fixed in https://github.com/sinhrks/ggfortify/pull/171. I am not sure about the other ones:
```
  1. Failure: autoplot.prcomp plots the desired components (@test-stats.R#509) 
  2. Failure: autoplot.prcomp plots the desired components (@test-stats.R#510) 
  3. Failure: autoplot.princomp plots the desired components (@test-stats.R#535) 
  4. Failure: autoplot.princomp plots the desired components (@test-stats.R#536) 
  5. Failure: autoplot.factanal plots the desired components (@test-stats.R#566) 
  6. Failure: autoplot.factanal plots the desired components (@test-stats.R#567)
```

**Comment by lselzer** [comment]
I'll look into it.

**Comment by lselzer** [comment]
Ok, now it passes. It cached a bug I introduced so good job for the tests.

**Comment by terrytangyuan** [comment]
Thanks!

**Pull request merged**